docs/_static/auth/v1.yaml

```diff
@@ -279,6 +279,67 @@ paths:
 
 components:
   schemas:
+    VerifiableCredential:
+      type: object
+      description: A credential according to the W3C and Nuts specs.
+      required:
+        - "@context"
```

Comment:
Many of these fields can also contain a string when there is only 1 value.

Reply:
Note that the `VerifiableCredential` type is a 1-on-1 copy from the `vcr` OpenAPI spec. Currently, this type is unused in Go as it's aliased to `vc.VerifiableCredential`. For now this feels like an "OK" solution.